inc/abstractitiltarget.class.php

```diff
@@ -2240,4 +2240,49 @@ protected function prepareUploadedFiles(array $data, $formanswer): array {
 
       return $data;
    }
+
+   public function isObjectTarget(): bool {
+      return true;
+   }
+
+   /**
+    * Find generated targets for this target and the given form answer
+    *
+    * @param PluginFormcreatorFormAnswer $formAnswer
+    * @return array
+    */
+   public static function findForFormAnswer(PluginFormcreatorFormAnswer $formAnswer): array {
```

Comment:
this method comes from the move of the SQL query from PluginFormcreatorFormAnswer::getGeneratedTargets()